R/bootstrap.R

```diff
@@ -1492,78 +1492,79 @@ downloadLink <- function(outputId, label="Download", class=NULL, ...) {
 #' Create an icon
 #'
 #' Create an icon for use within a page. Icons can appear on their own, inside
-#' of a button, or as an icon for a [tabPanel()] within a
-#' [navbarPage()].
-#'
-#' @param name Name of icon. Icons are drawn from the
-#'   [Font Awesome Free](https://fontawesome.com/) (currently icons from
-#'   the v5.13.0 set are supported with the v4 naming convention) and
-#'   [Glyphicons](https://getbootstrap.com/components/#glyphicons)
-#'   libraries. Note that the "fa-" and "glyphicon-" prefixes should not be used
-#'   in icon names (i.e. the "fa-calendar" icon should be referred to as
-#'   "calendar")
-#' @param class Additional classes to customize the style of the icon (see the
+#' of a button, or as an icon for a [tabPanel()] within a [navbarPage()].
+#'
+#' @param name The name of the icon. Icons are drawn from the [Font Awesome
+#'   Free](https://fontawesome.com/) (currently icons from the v5.13.0 set are
```

Comment:
The version number here is wrong -- but more generally, since the version can change independently of Shiny, perhaps we should link to the {fontawesome} package page, and list the version number there. Or link to the function that would implement https://github.com/rstudio/fontawesome/issues/30.

If those changes in fontawesome need to happen after the CRAN release, then we'll have to do something else in the meantime so that the incorrect version number is not shown here.

Reply:
How about we take out the version number for now? I can make changes in both packages' docs shortly after the release.

Reply:
Sure, that sounds reasonable.

Reply:
This is now changed.